coverage/coverage-utils.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Couldn't we just re-use `helpers.sh` instead of this file?

Reply:
Now that everything is working we can probably use helpers. I was trying to not break any other thing while I was testing the changes. The only difference in the functions is in the detection of terminals without color support.